example_configs/dokuwiki.md

```diff
@@ -0,0 +1,34 @@
+# Configuration for dokuwiki
+
+LDAP configuration is in ```/dokuwiki/conf/local.protected.php```:
+
+```
+<?php
+/**
+ * Protected settings
+ * LDAP configuration example
+ */
+$conf['useacl']         = 1;           //enable ACL
+$conf['authtype']       = 'authldap';  //enable this Auth plugin
+
+$conf['plugin']['authldap']['server']      = 'ldap://192.168.0.2:3890'; #IP of your lldap
```

Comment:
I would rather have the domain name than the IP here, so something like `ldap://lldap_server:3890`

Reply:
fixed